releases/release-8.5.1.md

```diff
@@ -0,0 +1,146 @@
+---
+title: TiDB 8.5.1 Release Notes
+summary: 了解 TiDB 8.5.1 版本的兼容性变更、改进提升，以及错误修复。
+---
+
+# TiDB 8.5.1 Release Notes
+
+发版日期：2025 年 x 月 x 日
+
+TiDB 版本：8.5.1
+
+试用链接：[快速体验](https://docs.pingcap.com/zh/tidb/v8.5/quick-start-with-tidb) | [生产部署](https://docs.pingcap.com/zh/tidb/v8.5/production-deployment-using-tiup) | [下载离线包](https://cn.pingcap.com/product-community/?version=v8.5.1#version-list)
+
+## 兼容性变更
+
+- note [#issue](https://github.com/pingcap/${repo-name}/issues/${issue-id}) @[贡献者 GitHub ID](https://github.com/${github-id})
+
+## 操作系统支持变更
+
+- 从 v8.5.1 起，TiDB 重新提供了对 CentOS Linux 7 的支持。如果你需要在 CentOS 7 上部署 TiDB 8.5 版本或将集群升级到 TiDB 8.5 版本，请部署或升级至 TiDB 8.5.1 或以上版本。
+    - 根据 [CentOS Linux EOL](https://www.redhat.com/en/blog/centos-linux-has-reached-its-end-life-eol)，CentOS Linux 7 的上游支持已于 2024 年 6 月 30 日终止。在 v8.4.0 DMR 和 v8.5.0 版本中，TiDB 暂停了对 CentOS 7 的支持，建议使用 Rocky Linux 9.1 及以上的版本。如果在使用 CentOS Linux 7 的情况下将 TiDB 升级到 v8.4.0 DMR 或 v8.5.0 版本，将导致集群不可用的风险。
+    - 尽管 TiDB v8.5.1 及以上版本支持 CentOS Linux 7，但由于 CentOS Linux 7 已经达到其生命周期的终止（EOL），强烈建议用户参考该系统的[官方声明和安全建议](https://www.redhat.com/en/blog/centos-linux-has-reached-its-end-life-eol)，考虑尽快迁移到 [TiDB 支持的操作系统版本](/hardware-and-software-requirements.md#操作系统及平台要求)，如 Rocky Linux 9.1 及以上版本。
+
+## 改进提升
+
++ TiDB
+
+    - note [#issue](https://github.com/pingcap/tidb/issues/${issue-id}) @[贡献者 GitHub ID](https://github.com/${github-id})
+    - note [#issue](https://github.com/pingcap/tidb/issues/${issue-id}) @[贡献者 GitHub ID](https://github.com/${github-id})
```

Comment:
```suggestion
    - note [#issue](https://github.com/pingcap/tidb/issues/${issue-id}) @[贡献者 GitHub ID](https://github.com/${github-id})
    - 增强了时间戳合法性检查。[#57786](https://github.com/pingcap/tidb/issues/57786) @[MyonKeminta](https://github.com/MyonKeminta)
```

Reply:
HOLD THIS: the PR is not merged into release-8.5 yet. It might be delayed to next version.

Reply:
Already merged